hermes/probe/probe.go

```diff
@@ -33,71 +33,64 @@ import (
 	cpmetrics "github.com/google/cloudprober/metrics"
```

Comment:
Renamed imports should go together. (Move it to line 38).

Reply:
Amended.